packages/@sanity/portable-text-editor/src/utils/vercelStegaCleanAll.ts

```diff
@@ -0,0 +1,24 @@
+import {vercelStegaSplit} from '@vercel/stega'
+
+/**
+ * This is a duplicate code from `@sanity/client/stega`
+ * Unfortunately, as it stands, the e2e process is pulling in the node version of `@sanity/client` and so we don't have access to the utility as it stands
+ * @todo remove once this utility is available in `@vercel/stega`
+ *
+ * Can take a `result` JSON from a `const {result} = client.fetch(query, params, {filterResponse: false})`
+ * and remove all stega-encoded data from it.
+ * @alpha
+ * @hidden
```

Comment:
Hiding it since hopefully this will be added at some point to the vercel functionality (we've requested it)